v3-nginx/Dockerfile

```diff
@@ -61,7 +61,7 @@ RUN set -eux; \
     ./build.sh; \
     git submodule init; \
     git submodule update; \
-    ./configure --with-yajl=/sources/yajl/build/yajl-${YAJL_VERSION}/ --with-geoip --with-pcre2; \
+    ./configure --with-yajl=/sources/yajl/build/yajl-${YAJL_VERSION}/ --with-geoip --with-pcre2="$(pcre2-config --prefix)"; \
```

Comment:
I wasn't able to find any documentation on this. From the original PR, it doesn't look like any options are supported.

Reply:
I just did this:
```
❯ ./configure --help | grep pcre2
  --with-pcre2=PATH       Path to pcre2 prefix or config script
```
So looks like it is indeed supported. You can see how it is used here: https://github.com/SpiderLabs/ModSecurity/blob/v3/master/build/pcre2.m4.

Maybe the solution is to create a PR against ModSecurity to use pcre2-config instead of pkg-config 🤔

Reply:
Looks like adding `libpcre-8` here: https://github.com/SpiderLabs/ModSecurity/blob/82f75dc0ce134c639df6c33bd610519bd4e90e42/build/pcre2.m4#L7 is a reasonable upstream fix.

Reply:
```
root@04e33073eeea:/# pkg-config --exists libpcre2-8
root@04e33073eeea:/# echo $?
0
root@04e33073eeea:/# pkg-config libpcre2-8 --modversion
10.36
```

Reply:
What we can do is patch it using that before building.

Reply:
Created https://github.com/SpiderLabs/ModSecurity/issues/2810